.github/ISSUE_TEMPLATE/dep-golang.md

```diff
@@ -54,6 +54,20 @@ SIG Release Slack thread:
   - [ ] test image
   - [ ] `.go-version` file
```

Comment:
should we add here to update in k/k as well?

Reply:
Added an update item there.